### PR TITLE
My Name Card

### PR DESCRIPTION
**Pull request by ghost**
Added image and card details.

### CODE REVIEW

File: utils/contributors.js [hunk 1]
```diff
@@ -33,6 +33,33 @@ const CONTRIBUTERS = [
         `,
         github: "https://github.com/subodhk01",
     },
+    // My Details
```
Comment (king-11) [inline utils/contributors.js]: no need for this comment

**Review by king-11 — CHANGES_REQUESTED** [review]
remove the comment rest LGTM 👍🏻

File: utils/contributors.js (hunk 1 above)
Comment (ghost) [reply utils/contributors.js]: Okay.

**Comment by ghost** [comment]
Removed the comment.

**Comment by king-11** [comment]
Can you also remove the blank line it isn't necessary

**Comment by ghost** [comment]
> Can you also remove the blank line it isn't necessary

Done

**Review by king-11 — APPROVED** [review]

**Comment by king-11** [comment]
Task completed successfully 🏅

**Pull request closed without merging**